client-templates/azure-repos/.env.sample

```diff
@@ -0,0 +1,28 @@
+# Guide how to get/create the token https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page
+# Scopes: Ensure Custom defined is selected and under Code select Read & write
+AZURE_REPOS_TOKEN=<token>
+
+# https://dev.azure.com/<organisation name> 
+AZURE_REPOS_ORG=<organisation name>
+
+# the host excluding scheme
+AZURE_REPOS_HOST=dev.azure.com
+
+# your unique broker identifier
+BROKER_TOKEN=<broker-token>
+
+# the url of your broker client (including scheme and port)
+BROKER_CLIENT_URL=<http://<broker client hostname:port>
```

Comment:
```suggestion
# BROKER_CLIENT_URL=
```
This suggestion is purely because it then matches what other samples have

Reply:
@yuliabaron suggested opposite, uncomment this😅 From user perspective, things that are commented out seems not important/must. But this is required setting

Reply:
Sounds fair, but can we raise a separate PR at the same time that updates all the other samples to match please?
I'd suggest the following..
`BROKER_CLIENT_URL=https://<BROKER_CLIENT_HOSTNAME>:<PORT>`

Reply:
Sure. Wil do